### DISCUSSION TITLE
When will the latest version of Unity release?

### DISCUSSION THREAD

**Discussion by wojiuxuihuan** [Facepunch/Facepunch.Steamworks#603] [Q&A] [answered]
Hi, It seems that the UnityPlugin haven't been touch for years, so many feature have been missing, will there be offical release, or we should manual pack it? I have try to repack them with VS2022, but after I import all the relate dll into Uniy, is Log's that the Win64 version is broken ( I think it may be something wrong with .net 6.0).

Also, In UGC section, I can't find the function to gain item's favorite state, or get all the favorite list by user, how to acheve this? Thanks!

**Comment by Mushe94** [comment, marked as answer]
You need to install Net 4.6 from Visual Studio Installer to build correctly.

**Comment by GFX47** [comment]
Any official release schedule though?

**Reply by vincent-savysoda** [reply]
Yes I would like to know too. I'm kinda half tempted to switch to Steamworks.Net atm coz they at least update their official release regularly and this is kinda blocking our MacOS release...

**Reply by vincent-savysoda** [reply]
@garrynewman any info on the timeline of next release would be greatly appreciated so we can make a decision on how to proceed here. Thank you.
